js/app.js

```diff
@@ -10,40 +9,68 @@ function Article (options) {
   this.publishedOn = options.publishedOn;
 }
 
-Article.prototype.toHtml = function() {
-  var $newArticle = $('article.template').clone();
-  $newArticle.removeClass('template');
-  if (!this.publishedOn) {
-    $newArticle.addClass('draft');
-  }
+Article.all = [];
 
-  $newArticle.attr('data-category', this.category);
-  $newArticle.attr('data-author', this.author);
+Article.prototype.toHtml = function() {
 
+  var appTemplate = $('#template').text();
 
+  var compileTemplate = Handlebars.compile(appTemplate);
 
-  $newArticle.find('.byline a').html(this.author);
-  $newArticle.find('.byline a').attr('href', this.authorUrl);
-  $newArticle.find('h1:first').html(this.title);
-  $newArticle.find('.article-body').html(this.body);
-  $newArticle.find('time[pubdate]').attr('datetime', this.publishedOn)
-  $newArticle.find('time[pubdate]').attr('title', this.publishedOn)
-  $newArticle.find('time').html('about ' + parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000) + ' days ago')
-  $newArticle.append('<hr>');
-  return $newArticle;
-}
+  var dataSource = {
+    title: this.title,
+    author: this.author,
+    authorUrl: this.authorUrl,
+    publishedOn: this.publishedOn,
+    body: this.body
+  }
 
-content.sort(function(a,b) {
-  return (new Date(b.publishedOn)) - (new Date(a.publishedOn));
-});
+  var html = compileTemplate(dataSource);
+
+this.daysAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000);
+this.publishStatus = this.publishedOn ? 'published ' + this.daysAgo + ' days ago' : '(draft)';
+
+ return compileTemplate(this);
+ };
+  // $('#byLine').append(html);
```

Comment:
@kmzurlocuva is this code commented out for future use/reference?  If not, best to get rid of it as it is tracked in previous commits and dirty's up the codebase a bit